spec/mem/atomic.md

```diff
@@ -0,0 +1,63 @@
+# MiniRust atomic memory
+
+This is a wrapper for a memory that distinguishes between non-atomic and atomic memory accesses.
```

Comment:
```suggestion
This is a wrapper for a memory that distinguishes between non-atomic and atomic memory accesses. For now atomicity is ignored; this will change in the future/
```